policy/variables.tf

```diff
@@ -78,6 +78,11 @@ variable "threshold" {
   description = "The value against which the specified statistic is compared."
 }
 
+variable "treat_missing_data" {
+  type        = "string"
+  description = "See http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html#alarms-and-missing-data"
```

Comment:
Looks like this value isn't available in the terraform documentation for this variable. [aws_cloudwatch_metric_alarm](https://www.terraform.io/docs/providers/aws/r/cloudwatch_metric_alarm.html#treat_missing_data)

Reply:
That's what happens when I try to rely on my memory :)